### PR TITLE
Update blitz.datadrive.sh

### PR DESCRIPTION
**Pull request by allyourbankarebelongtous**
Updates script for amd64 and nvme drives.

Changes blitz.datadrive.sh to accommodate nvme drive formats and amd64 processes.

Tests good on pi 4 and on intel nuc.

### CODE REVIEW

File: home.admin/config.scripts/blitz.datadrive.sh [hunk 1]
```diff
@@ -105,11 +105,17 @@ if [ "$1" = "status" ]; then
     fi
     lsblk -o NAME,SIZE -b | grep -P "[s|vn][dv][a-z][0-9]?" > .lsblk.tmp
     while read line; do
-
+
```
Comment (openoms) [inline home.admin/config.scripts/blitz.datadrive.sh]: nit: check the whitespaces

File: home.admin/config.scripts/blitz.datadrive.sh [hunk 2]
```diff
@@ -105,11 +105,17 @@ if [ "$1" = "status" ]; then
     fi
     lsblk -o NAME,SIZE -b | grep -P "[s|vn][dv][a-z][0-9]?" > .lsblk.tmp
     while read line; do
-
+      
       # cut line info into different informations
       testname=$(echo $line | cut -d " " -f 1 | sed 's/[^a-z0-9]*//g')
-      testdevice=$(echo $testname | sed 's/[^a-z]*//g')
-      testpartition=$(echo $testname | grep -P '[a-z]{3,5}[0-9]{1}')
+	  if [ $(echo $line | grep -c "nvme") = 0 ]; then
```
Comment (openoms) [inline home.admin/config.scripts/blitz.datadrive.sh]: identation: `if`, `else` and `fi` should fall in line.

File: home.admin/config.scripts/blitz.datadrive.sh [hunk 3]
```diff
@@ -193,7 +212,10 @@ if [ "$1" = "status" ]; then
     hddGigaBytes=0
     if [ "${hdd}" != "" ]; then
       hddBytes=$(fdisk -l /dev/$hdd | grep GiB | cut -d " " -f 5)
-      hddGigaBytes=$(echo "scale=0; ${hddBytes}/1024/1024/1024" | bc -l)
+	  if [ "${hddBytes}" = "" ]; then
+	    hddBytes=$(fdisk -l /dev/$hdd | grep TiB | cut -d " " -f 5)
+	  fi
+      ddGigaBytes=$(echo "scale=0; ${hddBytes}/1024/1024/1024" | bc -l)
```
Comment (openoms) [inline home.admin/config.scripts/blitz.datadrive.sh]: typo
```suggestion
      hddGigaBytes=$(echo "scale=0; ${hddBytes}/1024/1024/1024" | bc -l)
```

**Comment by openoms** [comment]
Thanks for the improvements, left some comments about a typo and few indentation issues. 
We use 2 spaces for nested if; then structures.

File: home.admin/config.scripts/blitz.datadrive.sh (hunk 1 above)
Comment (allyourbankarebelongtous) [reply home.admin/config.scripts/blitz.datadrive.sh]: Cleaned up

**Comment by openoms** [comment]
That's much better! LGTM

**Review by openoms — APPROVED** [review]

**Comment by allyourbankarebelongtous** [comment]
> That's much better! LGTM

Thanks! I don't know why my text editor at home has good indents but when it gets on here everything is skewed. Probably a tab space setting or something.

**Comment by rootzoll** [comment]
Going to merge for v1.9 ReleaseCandidate.

@openoms how much of this related issue #2228 is covered with this PR?

**Comment by allyourbankarebelongtous** [comment]
> Going to merge for v1.9 ReleaseCandidate.
> 
> @openoms how much of this related issue #2228 is covered with this PR?

At least for me at first glance it should solve it. The script now looks for nvme in front of the device name and if it's there it parses the data and partitions using nvme formats instead of sata. It works on the nuc I have, and it also works on my pi, but the pi doesn't have an nvme drive so I technically can't test it with a pi AND nvme at the same time.

**Pull request merged**